static/templates/message_feed_errors.hbs

```diff
@@ -5,7 +5,7 @@
         Some older messages are unavailable.
         <z-link>Upgrade your organization</z-link>
         to access your full message history.
-        {{#*inline "z-link"}}<a href="/plans/" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
+        {{#*inline "z-link"}}<a href="https://zulip.com/plans/" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
```

Comment:
Same—I think we’d never expect this to be shown on a self-hosted server (unless the admin manually edited the realm’s plan in the database, but they’d have no reason to do that).